class-00/lab-c/starter-code/styles/modules.css

```diff
@@ -0,0 +1,8 @@
+nav ul {
```

Comment:
Is this is how you want the nav lists to always show on every page? Generally, I think that the default nav items styling should go in layout. Perhaps if the defaults were being overwritten for certain pages or views, or in some contexts, it would make sense to have these overrides in modules.